contracts/deploy/foreign/arbitrum.js

```
@@ -22,18 +22,20 @@ async function deployForeignProxy({ deploy, from, parameters, homeNetworkName, h
   const surplus = eth("0.03"); // The surplus will be automatically reimbursed when the dispute is created.
   const l2GasLimit = 1500000; // Gas limit for a tx on L2.
   const gasPriceBid = 1000000000; // x10000 bid of random arb sepolia tx. Gas price * gasLimit will result in additional 0.0015 eth fee for automatic-redeem on L2. The surplus will be reimbursed.
+  const chainId = await getChainId();
```

ESLint

[error] 'getChainId' is not defined.

(no-undef)